imjoy/workers/python3_client.py

```diff
@@ -123,7 +123,7 @@ def __init__(self, id=None):
         """Set up client instance."""
         super().__init__(id)
         self.loop = asyncio.get_event_loop()
-        self.janus_queue = janus.Queue(loop=self.loop)
+        self.janus_queue = janus.Queue()
```

Comment:
The queue must be instantiated within the asyncio event loop. Otherwise it doesn't work getting the current event loop.

Reply:
The loop is already there right? Or do you mean we need to run the loop before instantiate it?

So why our tests passed?

Reply:
We're instantiating this in the main function. Is that main function called within the event loop?

Reply:
We're only testing this with CI on Python 3.6. I'm pretty sure it will fail in Python 3.7.

Reply:
I think you are right after reading more about the changes.

It works because also in jupyter the loop is already running.

However, I think it's not necessary to run exactly inside the loop call right, we need to only make sure the loop is running when we instantiate the queue, right?

Well, I think for now I would revert the changes and pin janus to a 0.4.0, since we will change this repo soon to use imjoy-rpc library.

Reply:
`asyncio.get_running_loop` can only be called within a coroutine or callback, ie inside the event loop.

https://docs.python.org/3/library/asyncio-eventloop.html#asyncio.get_running_loop